test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -47,12 +58,17 @@ public void elementsAreUnique() throws Exception {
         assertAreUnique(1);
         assertAreUnique("");
         assertAreUnique("abc");
+        assertAreUnique(new Integer(0));
 
         // all objects unique
         assertAreUnique("abc", "ab", "a");
         assertAreUnique(1, 2);
+        assertAreUnique(new Object(), new Object());
 
         // some identical objects
+        String a = "asdf";
+        String b = "asdf";
+        assertNotUnique(a, b);
```

Comment:
The expectation was to write a test case method. 

Note the naming convention for test methods (the first part is feature/method under test), second is the testing scenario and the third is the expected behaviour. 
E.g. isAnyNull_emptyList_falseReturned
Part 2 and 3 of the test name can be removed to indicate an increased test scope